modules/mbedtls/CMakeLists.txt

```diff
@@ -35,6 +35,10 @@ if(CONFIG_ARCH_POSIX AND CONFIG_ASAN AND NOT CONFIG_64BIT)
       "${OPTIMIZE_FOR_SPEED_FLAG}")
 endif ()
 
+if(CONFIG_MBEDTLS_ZEPHYR_ENTROPY AND NOT CONFIG_ENTROPY_HAS_DRIVER)
+  message(WARNING "No entropy device on the system, using fake entropy source!")
+endif()
+
```

Comment:
this is actually inside another `if()` (twice actually), so it ought to be indented.

But I can see the whole file is messed up, so also acceptable to let it be as-is, and do a file wide fix later.

Reply:
The indentation is now fixed.